src/layouts/SiteDashboard/SiteDashboard.tsx

```diff
@@ -193,9 +200,67 @@ export const SiteDashboard = (): JSX.Element => {
           {/* Right column */}
           <Box w="40%">
             <VStack spacing="1.25rem">
+              {/* Site launch display card */}
+              {isUserUsingSiteLaunchFeature(siteName) &&
+                siteLaunchStatus !== "LAUNCHED" && (
+                  <Box w="100%">
+                    <DisplayCard variant="content">
+                      <Skeleton isLoaded={!isSiteLaunchLoading}>
+                        <DisplayCardHeader
+                          button={
+                            <Link
+                              variant="link"
+                              textStyle="subhead-1"
+                              color="text.title.brand"
+                              marginRight="0.75rem"
+                              href="./siteLaunchPad"
+                            >
```

Comment:
this `Link` isn't actually what you want - what this does is perform a hard fetch (when you click on it in the storybook, it fails for this reason). 

we want to use the `Link` from `react-router-dom` and `as` the ogp `Link` (see [here](https://plainenglish.io/blog/const-vs-function-for-react-functional-components)) for details

Reply:
sure I have used `Link` but to be clear, I do want to perform a hard fetch to the newer url. Ie, the storybook link _should_ fail as the sitelaunchPad Stuff is not introduced in this PR (its introduced in #1297 instead)!